Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -2397,6 +2398,9 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>, publ
 
 #if ENABLE(ASYNC_SCROLLING) && PLATFORM(COCOA)
     void sendScrollPositionChangedForNode(std::optional<WebCore::FrameIdentifier>, WebCore::ScrollingNodeID, const WebCore::FloatPoint& scrollPosition, std::optional<WebCore::FloatPoint> layoutViewportOrigin, bool syncLayerPosition, bool isLastUpdate);
+    void sendScrollingTreeNodeScrollbarVisibilityDidChange(std::optional<WebCore::FrameIdentifier>, WebCore::ScrollingNodeID, WebCore::ScrollbarOrientation, bool isVisible);
```

Comment:
This doesn't seem very extensible. We can't add unique functions for every different message that we want to send.

Why not add a "nodeID to FrameIdentifier" helper and use that instead?

Reply:
Right, changed this to make `sendToProcessContainingFrame` public so it can be used directly in `RemoteScrollingCoordinatorProxy`.